docs/uma-tokenholders/general-kpi.md

```diff
@@ -0,0 +1,144 @@
+
+## Main Launch Parameters
+
+When launching contracts with the [LSP launch package](https://github.com/UMAprotocol/launch-lsp) main parameters to consider for generalized KPI options are listed below:
+
+* `expirationTimestamp`: UNIX timestamp when the KPI options should expire that should be aligned with your program objectives.
+* `collateralPerPair`: This sets the maximum amount of collateral tokens that could be paid out per one KPI options token. The amount should be scaled up by 10^18 when passed as a parameter to the [LSP launch script](https://github.com/UMAprotocol/launch-lsp/blob/master/index.js).
+* `priceIdentifier`: Should be set to `General_KPI`.
+* `customAncillaryData`: This should include the KPI options target metric, data sources and calculation methods as detailed in the [Ancillary Data](/uma-tokenholders/general-kpi.md#ancillary-data-parameters) section.
+* `optimisticOracleLivenessTime`: Time in seconds, the proposed expiration price can be disputed. This generally should be at least a couple of hours for KPI options but might require longer time for more complex KPI options target metric calculations.
+* `fpl`: Choose the type of financial product library (FPL) that will determine the payout function based on resolved target KPI metric. Most applicable FPLs for KPI options might be:
+  * `BinaryOption` to select [Binary Options](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/BinaryOptionLongShortPairFinancialProductLibrary.sol) type that would either pay out nothing or maximum amount (set in `collateralPerPair`) to KPI options token holders depending on whether the target metric threshold is met.
+  * `Linear` to select [Linear](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/LinearLongShortPairFinancialProductLibrary.sol) type that would pay out nothing when lower bound of target metric threshold is not met, a maximum amount (set in `collateralPerPair`) when upper bound of target metric threshold is met, and proportional payout when target metric is between its lower and upper bounds.
+
+## Ancillary Data Parameters
+
+Below are the parameters that are required when using the generalized KPI options. A short description of the parameters and a few examples illustrate the flexibility in using the generalized KPI option identifier. Please consult [UMIP-117](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-117.md) for ancillary data formatting syntax.
+
+### Metric
+
+A short description and units of the KPI or metric you wish to incentivize. The metric is what is considered the actual number when setting your KPI options. This description should be short and to the point.
+
+#### Metric Examples
```

Comment:
These seem too broad for actual usage, so might not be very good examples. Should probably specify an example protocol/liquidity pool etc.

Reply:
Updated to UMA specific examples